tools/release_suite_lists/beta13_list

```diff
@@ -0,0 +1,64 @@
+aAPKLIST="sampleapp-android-tests
```

Comment:
Are these lists targeting for branches of the repository? For example, this list `beta13_list` is for https://github.com/crosswalk-project/crosswalk-test-suite/tree/Crosswalk-13; `beta14_list` for Crosswalk-14; and `release_list` for the master branch?